src/main/controllers/index.ts

```diff
@@ -13,7 +13,7 @@ export * as soleTraderNationalityController from "./features/SoleTrader/soleTrad
 export * as typeOfBusinessController from "./features/SoleTrader/typeOfBusinessController";
 export * as OtherTypeOfBusinessController from "./features/SoleTrader/otherTypeOfBusinessController";
 export * as healthCheckController from "./healthCheckController";
-export * as nameRegisteredWithAmlController from "./features/limited/nameRegisteredWithAmlController";
-export * as businessMustbeAmlRegisteredController from "./features/limited/businessMustbeAmlRegisteredController";
-export * as whatIsTheBusinessNameController from "./features/unincorporated/whatIsTheBusinessNameController";
-export * as companyInactiveController from "./features/limited/companyInactiveController";
+export * as nameRegisteredWithAmlController from "./features/Limited/nameRegisteredWithAmlController";
```

Comment:
'limited' should be used instead of 'Limited', as per updated folder structure

Reply:
Not sure why is it still showing "L", its actually "l" only